galvan/src/main/java/org/terracotta/testing/master/FileHelpers.java

```diff
@@ -80,8 +80,11 @@ public FileVisitResult postVisitDirectory(Path dir, IOException exc) throws IOEx
   public static String createTempCopyOfDirectory(ILogger logger, String targetParentDirectoryString, String newDirectoryName, String sourceDirectoryString) throws IOException {
     FileSystem fileSystem = FileSystems.getDefault();
     Path targetParentDirectory = fileSystem.getPath(targetParentDirectoryString);
+    logger.log(" Target directory: " + targetParentDirectoryString);
     Assert.assertTrue(targetParentDirectory.toFile().isDirectory());
+    logger.log(" Source directory: " + sourceDirectoryString);
     Path sourceDirectory = fileSystem.getPath(sourceDirectoryString);
+    Assert.assertTrue(sourceDirectory.toFile().exists());
```

Comment:
It does, but I split this out specifically so I could see what the failing assert was (there is some problem where sometimes the kit isn't installed before running the test).  It was effectively the same problem as `assert(clause1 && clause2)` being made more meaningful by splitting to 2 asserts.